src/layout/layout.module.css

```diff
@@ -19,22 +21,37 @@
   max-width: 50rem;
   margin: 0 auto;
 }
+
 .header__logo img {
   display: block;
   margin: 1.2rem;
   height: 0.9rem;
 }
+
 .header__nav {
   flex: 1;
 }
-.header__nav ul {
+
+.header__nav__ul {
   display: flex;
   align-content: center;
   justify-content: center;
   list-style: none;
   margin: 0;
   padding: 0;
+  transition: 0.4s;
 }
+
+.header__nav__ul li:last-child {
+  display: none;
+}
+
+.header__nav__ul a {
+  font-family: Recoleta;
+  color: #333333;
+  font-size: 14px;
```

Comment:
Should use `rem`s. Not for the benefit of relative sizing, but more for consistency throughout the code.

Reply:
done^^